node-scripts/spec/pipeline3/impl/dockerComposeGenericMaven.spec.js

```diff
@@ -0,0 +1,174 @@
+"use strict";
+
+describe("Docker Compose Maven implementation", function() {
+  // deps
+  const path = require("path");
+  const config = require(path.resolve("src/utils/config"));
+  const cst = require(path.resolve("src/const"));
+  const dockerCompose = require(path.resolve(
+    "src/pipeline3/impl/dockerComposeGenericMaven"
+  ));
+
+  const scripts = require(path.resolve(
+    "src/" + config.getJobNameForPipeline3() + "/scripts"
+  ));
+
+  var instanceDef = {
+    uuid: "336af1ee-90a1-4d1b-baaf-db12c84deec0",
+    name: "cambodia1",
+    type: "dev",
+    group: "tlc",
+    deployment: {
+      hostDir: "/var/docker-volumes/",
+      type: "dockerComposeGenericMaven",
+      composePlugin: true,
```

Comment:
`composePlugin: true,`
should be
`composePlugin: "true",`

Reply:
This is a boolean, and it is supported in JSON. Why make it a String which we will then need to parse?

Reply:
OK. In that case, there are other places where the code assumes a string.
We should change that so that we stay consistent. I'll prepare a follow-up PR.